rxjava-2-examples/src/main/java/io/vertx/example/reactivex/http/client/reduce/Client.java

```diff
@@ -3,9 +3,9 @@
 import io.reactivex.Flowable;
 import io.reactivex.Maybe;
 import io.vertx.core.Launcher;
+import io.vertx.core.buffer.Buffer;
```

Comment:
Todo: document Buffer package change in RX due to Buffer being a data object in 5